webapp-django/crashstats/supersearch/forms.py

```diff
@@ -30,34 +30,22 @@ def make_restricted_choices(sequence, exclude=None):
 class SearchForm(forms.Form):
     """Handle the data populating the search form"""
 
-    def __init__(
-        self,
-        all_fields,
-        products,
-        product_versions,
-        current_platforms,
-        user,
-        *args,
-        **kwargs
-    ):
+    def __init__(self, all_fields, products, product_versions, platforms, user, *args, **kwargs):
         super(self.__class__, self).__init__(*args, **kwargs)
 
         self.all_fields = all_fields.copy()
 
-        # Default values loaded from a database.
-        product_names = list(set(x['product_name'] for x in products))
+        # Generate default values
```

Comment:
`products` is passed in as a flat list of product names, so we don't have to do anything with it. I also changed `platforms` in the same way.